yaml/gateway-svc.yml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   type: NodePort
   ports:
-    - port: 8080
+    - port: 808
```

Comment:
808 is a very odd low port number. I think 8081 might be better or 8088

Reply:
This was in error. Correcting :+1:

Reply:
I'm seeing the "How Has This Been Tested?" field empty. This is a high impact change, please could you test independently with the YAML and helm approach?

Reply:
As commented, I have tested with Helm. I am testing YAML now :+1: